libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/view/MapboxManeuverView.kt

```diff
@@ -249,29 +299,39 @@ class MapboxManeuverView : ConstraintLayout {
         binding.upcomingManeuverRecycler.visibility = visibility
     }
 
+    /**
+     * Invoke the method to enable/disable showing the list of upcoming maneuvers.
+     * Note: The API doesn't maintain the state of this property. If the device undergoes
+     * screen rotation changes make sure to invoke this method again at appropriate place.
+     * @param toggle Boolean either true/false to enable or disable showing the upcoming maneuver
+     */
+    fun toggleUpcomingManeuverRendering(toggle: Boolean) {
+        isShowUpcomingManeuver = toggle
+    }
```

Comment:
nit: "toggle" methods typically do not take an argument, how about:
```suggestion
    var upcomingManeuverRenderingEnabled = true
        set(value) {
            if (value != field) {
                if (!value && binding.upcomingManeuverRecycler.visibility == VISIBLE) {
                    updateUpcomingManeuversVisibility(View.GONE)
                }
            }
            field = value
        }
```

We also need to handle hiding the component if it's disabled anyway.